renovate.json

```diff
@@ -14,16 +14,15 @@
   ],
   "packageRules": [
     {
-      "matchPackagePatterns": ["^io.grpc:grpc-"],
-      "groupName": "gRPC packages"
+      "matchPackagePrefixes": ["com.google"],
+      "enabled": false
     },
     {
-      "matchPackagePatterns": ["^com.google.protobuf:protobuf-"],
-      "groupName": "Protobuf packages"
-    },
-    {
-      "matchPackagePatterns": ["^io.opencensus:opencensus-"],
-      "groupName": "OpenCensus packages"
+      "matchPackageNames": [
```

Comment:
Documentation of matchPackageNames https://docs.renovatebot.com/configuration-options/#matchpackagenames